docs/tutorials/optimizers_conditionalgradient.ipynb

```diff
@@ -0,0 +1,558 @@
+{
```

Comment:
<p>Could you further describe why this comparison of Frobenius norm is important.</p><p> </p><p>Also, could you plot the validation losses against each other.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/590/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
Still need to write something about this.